diff -ruN openssl-1.0.1j.orig/crypto/ui/ui_openssl.c openssl-1.0.1j/crypto/ui/ui_openssl.c
--- openssl-1.0.1j.orig/crypto/ui/ui_openssl.c	2014-10-15 14:53:39.000000000 +0200
+++ openssl-1.0.1j/crypto/ui/ui_openssl.c	2014-11-05 13:10:13.574510723 +0100
@@ -410,7 +410,7 @@
 	return 1;
 	}
 
-static volatile sig_atomic_t intr_signal;
+static volatile int intr_signal;
 #endif
 
 static int read_string_inner(UI *ui, UI_STRING *uis, int echo, int strip_nl)
