From 547c69c81af05bd010ef968217dd2bf5be2062ae Mon Sep 17 00:00:00 2001
From: Lars Schmertmann <Lars.Schmertmann@governikus.de>
Date: Fri, 12 Feb 2021 13:15:00 +0100
Subject: Adjust iOS target

---
 Configurations/15-ios.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git x/Configurations/15-ios.conf y/Configurations/15-ios.conf
index f404077fda..b6eaa562d1 100644
--- x/Configurations/15-ios.conf
+++ y/Configurations/15-ios.conf
@@ -24,7 +24,7 @@ my %targets = (
     "ios64-xcrun" => {
         inherit_from     => [ "ios-common", asm("aarch64_asm") ],
         CC               => "xcrun -sdk iphoneos cc",
-        cflags           => add("-arch arm64 -mios-version-min=7.0.0 -fno-common"),
+        cflags           => add("-arch arm64 -mios-version-min=11.0 -fno-common -fembed-bitcode"),
         bn_ops           => "SIXTY_FOUR_BIT_LONG RC4_CHAR",
         perlasm_scheme   => "ios64",
     },
-- 
2.32.0

