From 5b973bedaed49a7826231701adedf32c276e3c74 Mon Sep 17 00:00:00 2001
From: Lars Schmertmann <Lars.Schmertmann@governikus.de>
Date: Tue, 19 Jan 2021 17:07:51 +0100
Subject: android shlib_variant

---
 Configurations/15-android.conf | 5 +++++
 1 file changed, 5 insertions(+)

diff --git x/Configurations/15-android.conf y/Configurations/15-android.conf
index fd5cd3f478..8a8683b4e4 100644
--- x/Configurations/15-android.conf
+++ y/Configurations/15-android.conf
@@ -190,6 +190,7 @@ my %targets = (
         bn_ops           => sub { android_ndk()->{bn_ops} },
         bin_cflags       => "-pie",
         enable           => [ ],
+        shared_extension => '.so',
     },
     "android-arm" => {
         ################################################################
@@ -221,11 +222,13 @@ my %targets = (
         #
         inherit_from     => [ "android", asm("armv4_asm") ],
         bn_ops           => add("RC4_CHAR"),
+        shlib_variant    => '_armeabi-v7a',
     },
     "android-arm64" => {
         inherit_from     => [ "android", asm("aarch64_asm") ],
         bn_ops           => add("RC4_CHAR"),
         perlasm_scheme   => "linux64",
+        shlib_variant    => '_arm64-v8a',
     },
 
     "android-mips" => {
@@ -253,11 +256,13 @@ my %targets = (
         CFLAGS           => add(picker(release => "-fomit-frame-pointer")),
         bn_ops           => add("RC4_INT"),
         perlasm_scheme   => "android",
+        shlib_variant    => '_x86',
     },
     "android-x86_64" => {
         inherit_from     => [ "android", asm("x86_64_asm") ],
         bn_ops           => add("RC4_INT"),
         perlasm_scheme   => "elf",
+        shlib_variant    => '_x86_64',
     },
 
     ####################################################################
-- 
2.32.0

