From c97e9531a9da0ad5ae3bfb7cec90b03475a58a76 Mon Sep 17 00:00:00 2001
From: Lars Schmertmann <Lars.Schmertmann@governikus.de>
Date: Fri, 12 Feb 2021 13:15:00 +0100
Subject: Adjust iOS target

---
 Configurations/15-ios.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git x/Configurations/15-ios.conf y/Configurations/15-ios.conf
index 54d37f63f4..7e411b2e3a 100644
--- x/Configurations/15-ios.conf
+++ y/Configurations/15-ios.conf
@@ -25,7 +25,7 @@ my %targets = (
     "ios64-xcrun" => {
         inherit_from     => [ "ios-common" ],
         CC               => "xcrun -sdk iphoneos cc",
-        cflags           => add("-arch arm64 -mios-version-min=7.0.0 -fno-common"),
+        cflags           => add("-arch arm64 -mios-version-min=11.0 -fno-common"),
         bn_ops           => "SIXTY_FOUR_BIT_LONG RC4_CHAR",
         asm_arch         => 'aarch64',
         perlasm_scheme   => "ios64",
