From f63dce0a614ad26176dfbf4069264a4ec36c11b1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Klitzing?= <aklitzing@gmail.com>
Date: Tue, 20 Nov 2018 16:05:20 +0100
Subject: [PATCH] android shlib_variant

---
 Configurations/15-android.conf | 2 ++
 1 file changed, 2 insertions(+)

diff --git x/Configurations/15-android.conf y/Configurations/15-android.conf
index 7b496a4529..559e11e27f 100644
--- x/Configurations/15-android.conf
+++ y/Configurations/15-android.conf
@@ -170,6 +170,8 @@ my %targets = (
         bn_ops           => sub { android_ndk()->{bn_ops} },
         bin_cflags       => "-pie",
         enable           => [ ],
+        shlib_variant    => '-gov',
+        shared_extension => '.so',
     },
     "android-arm" => {
         ################################################################
-- 
2.20.1

