From 85caf189e67e25335589dfe7ab52aff10cf79c5b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andr=C3=A9=20Klitzing?= <aklitzing@gmail.com>
Date: Wed, 25 Sep 2019 17:17:45 +0200
Subject: [PATCH] Remove unused plugins from the build

---
 src/imports/imports.pro | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git x/qtdeclarative/src/imports/imports.pro y/qtdeclarative/src/imports/imports.pro
index 997388302..7007f100a 100644
--- x/qtdeclarative/src/imports/imports.pro
+++ y/qtdeclarative/src/imports/imports.pro
@@ -4,10 +4,8 @@ QT_FOR_CONFIG += qml-private
 SUBDIRS += \
     builtins \
     qtqml \
-    models \
-    labsmodels
+    models
 
-qtConfig(qml-itemmodel): SUBDIRS += folderlistmodel
 qtConfig(qml-worker-script): SUBDIRS += workerscript
 qtHaveModule(sql): SUBDIRS += localstorage
 qtConfig(settings): SUBDIRS += settings
@@ -25,7 +23,6 @@ qtHaveModule(quick) {
     qtConfig(quick-shadereffect): SUBDIRS += wavefrontmesh
 
     qtHaveModule(testlib): SUBDIRS += testlib
-    qtConfig(systemsemaphore): SUBDIRS += sharedimage
     qtConfig(quick-particles): \
         SUBDIRS += particles
 
-- 
2.23.0

