From c7e2123821cd3eb66ee0330567acfa97a24a9b40 Mon Sep 17 00:00:00 2001
From: Lars Schmertmann <Lars.Schmertmann@governikus.de>
Date: Tue, 13 Feb 2024 07:34:17 +0100
Subject: Increase NFC timeout on Android

---
 src/nfc/qnearfieldtarget_android.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git x/qtconnectivity/src/nfc/qnearfieldtarget_android.cpp y/qtconnectivity/src/nfc/qnearfieldtarget_android.cpp
index 0dcd602b7400bc9ec57fd3a4f0536f014051d5e8..88155fa376496b7f949bd5a6ca0798d8ef871325 100644
--- x/qtconnectivity/src/nfc/qnearfieldtarget_android.cpp
+++ y/qtconnectivity/src/nfc/qnearfieldtarget_android.cpp
@@ -453,7 +453,7 @@ bool QNearFieldTargetPrivateImpl::connect()
     if (connected)
         return true;
 
-    setCommandTimeout(2000);
+    setCommandTimeout(3000);
     methodId = env.findMethod<void>(tagTech.objectClass(), "connect");
     if (!methodId)
         return false;
